osu.Game/Online/Matchmaking/Events/MatchmakingJumpEvent.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using MessagePack;
+using osu.Game.Online.Multiplayer;
+
+namespace osu.Game.Online.Matchmaking.Events
+{
+    [Serializable]
+    [MessagePackObject]
+    public class MatchmakingJumpEvent : MatchServerEvent
```

Comment:
Rather than having one request class/type per action, don't you think having a `MatchmakingUserActionEvent` with an enum would scale better?

Reply:
I think that depends on the type of data/events, which isn't well defined at this point.

For example consider:
- An event to display cursors in the room, `CursorFrame { Position }` (I started initial work on this but would like to discuss if it should exist before continuing)
- An event to paste a sticker, `Sticker { Position, Type }`.
- Can you think of any more events? Would probably need to discuss.

Reply:
I'm just thinking the very basic in allowing for future scope, which is that there's likely going to be more than one event which has no parameters and is just a "user did thing". So using this for the jump would be a good idea, with very minimal implementation overhead (just an enum with a single `Jump` for now).